.devcontainer/base-image/Dockerfile

```diff
@@ -1,17 +1,20 @@
-# Some of this is shared with /Dockerfile
+# syntax=docker/dockerfile:1.4
 
 FROM mcr.microsoft.com/devcontainers/rust:1-1-bullseye
 
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
     cmake \
-    && rm -rf /var/lib/apt/lists/* \
-    && sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d -b /usr/local/bin
```

Comment:
I do think we need this in the image though, or none of our `task` command will work in the final image?

Reply:
Good point.
The Task is installed on the user side via the Dev Container Feature (that I mantain), the advantage of installing via the Feature is that the Task's VSCode extension and shell completion are also configured.
https://github.com/PRQL/prql/blob/28b9653b4d0a52427dde2269d2d5826d09a695f3/.devcontainer/devcontainer.json#L8

Reply:
OK great!